packages/vite/src/node/build.ts

```diff
@@ -93,10 +93,13 @@ export interface BuildOptions {
    */
   cssCodeSplit?: boolean
   /**
-   * Whether to generate sourcemap
+   * If true, a separate sourcemap file will be created. If "inline", the
+   * sourcemap will be appended to the resulting output file as data URI.
+   * "hidden" works like true except that the corresponding sourcemap
+   * comments in the bundled files are suppressed.
```

Comment:
Apply common doc style

```suggestion
   * If `true`, a separate sourcemap file will be created. If 'inline', the
   * sourcemap will be appended to the resulting output file as data URI.
   * 'hidden' works like `true` except that the corresponding sourcemap
   * comments in the bundled files are suppressed.
```